grid-layout/src/test/scala/react/gridlayout/PackageTests.scala

```diff
@@ -24,7 +24,7 @@ object PackageTests extends TestSuite {
       val layout = ReactGridLayout(200, <.div("Abc"))
       ReactTestUtils.withRenderedIntoDocument(layout) { m =>
         val html =
-          """<div class="react-grid-layout" style="height: 170px;"></div>""".stripMargin
+          """<div class="react-grid-layout" style="height: 10px;"></div>""".stripMargin
```

Comment:
@cquiroz is it weird/wrong that this changed?

Reply:
Not really, upstreams changes this stuff sometimes, it is a big change though

Reply:
Thanks. I'm not sure if it's related to the dependency version. I just fixed it to match what's in the old repo.